### DISCUSSION TITLE
[Q&A] Virtual Machine Clone use the same agent on server

### DISCUSSION THREAD

**Discussion by nykroy** [glpi-project/glpi-agent#439] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

I have two glpi intances that one is a virtual machine clone of production instance. if i try to inventory the dev instance on prod instance on glpi-inventory plugin i see that the agent of production have inventory the virtual machine of dev. 
the hostname is not the same on prod an dev. 
![image](https://github.com/glpi-project/glpi-agent/assets/7745881/436ad9c9-68f6-45e7-a4a6-b3b7c5842a1c)
VL0870 is dev instance and vp0488 es prod instance

**Comment by g-bougard** [comment]
Hi @nykroy 

on the dev instance in the vm, you should remove the `GLPI-Agent.dump` file you will find in the glpi-agent var dir (generally `/var/lib/glpi-agent` on linux and `C:\Program Files\GLPI-Agent\var` on windows). Then restart the service and the agent will generate a new deviceid.

**Reply by nykroy** [reply]
thansk!!!!!!
